config-editor/config-editor-ui/src/app/model/config-model.ts

```diff
@@ -291,3 +291,18 @@ export interface ErrorDialog {
   icon_name: string,
   icon_color: string,
 }
+
+export interface TestConfigSpec {
+  name: string;
+  test_schema: JSONSchema7;
+  config_testing: boolean;
+  test_case_testing: boolean;
+  release_testing: boolean;
+  incomplete_result: boolean;
+}
+
+export interface TestConfigSpecTesters {
+  config_testers: TestConfigSpec[];
+}
+
+export const DEFAULT_CONFIG_TESTER_NAME = "default";
```

Comment:
is is not necessary to store this name in ui and use this constant. it is provided by backend and ui should use name of testers form backend